delivery_package_gs1_barcode_report/reports/stock_report_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+        <record id="action_report_picking_packages" model="ir.actions.report">
+            <field name="name">SSCC Labels</field>
```

Comment:
The name of this report should be something like "Package label including GS1 barcodes" because there's not only an SSCC code on there but also other GS1 barcodes that specify the weight etc.